### PR TITLE
For sound-protocol, filter all to SoundEditionCreated events and log the SoundEditionV1 contract addresses #295

### PR DESCRIPTION
**Pull request by Kunkka0822**

### CODE REVIEW

**Review by TimDaub — CHANGES_REQUESTED** [review]
- [ ] please rename from "newsound" to "sound-protocol" as it's what they call it too.

File: src/strategies/newsound-filter-contracts/transformer.mjs [hunk 1]
```diff
@@ -0,0 +1,80 @@
+// @format
+import { decodeLog } from "eth-fun";
+
+import logger from "../../logger.mjs";
+import { parseJSON } from "../../utils.mjs";
+
+export const name = "newsound-filter-contracts";
```
Comment (TimDaub) [inline src/strategies/newsound-filter-contracts/transformer.mjs:7]:
```suggestion
export const name = "sound-protocol-filter-contracts";
```

File: test/strategies/newsound-filter-contracts/transformer_test.mjs [hunk 2]
```diff
@@ -0,0 +1,239 @@
+//@format
+import test from "ava";
+
+import { onLine } from "../../../src/strategies/newsound-filter-contracts/transformer.mjs";
```
Comment (TimDaub) [inline test/strategies/newsound-filter-contracts/transformer_test.mjs:4]:
```suggestion
import { onLine } from "../../../src/strategies/sound-protocol-filter-contracts/transformer.mjs";
```

File: test/strategies/newsound-filter-contracts/transformer_test.mjs [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+test("newsound-filter-contracts transformer", (t) => {
+  const editionCreatedSelector =
+    "0x405098db99342b699216d8150e930dbbf2f686f5a43485aed1e69219dafd4935";
+  const res0 = onLine(JSON.stringify(snapshot0), editionCreatedSelector);
+  const res1 = onLine(JSON.stringify(snapshot1), editionCreatedSelector);
+
+  t.deepEqual(JSON.parse(res0), {
+    "0xdc01dc4ccfca6238ec697c2cd29a1408c675d79e": {
+      name: "newsound",
```
Comment (TimDaub) [inline test/strategies/newsound-filter-contracts/transformer_test.mjs:235]:
```suggestion
      name: "sound-protocol",
```

Comment (Kunkka0822) [reply test/strategies/newsound-filter-contracts/transformer_test.mjs:235]: okay

**Pull request merged**